### PR TITLE
[AIRFLOW-373] Enhance CLI variables functionality

### PR DESCRIPTION
**Pull request by normster**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-373

Testing Done:
- Extended CLI unit tests

Add export/import to/from json file option for CLI variable command.
Add delete variable option for CLI variable command.

### CODE REVIEW

File: tests/core.py [hunk 1]
```diff
@@ -906,7 +906,52 @@ def test_variables(self):
             'variables', '-g', 'baz', '-d', 'bar']))
         cli.variables(self.parser.parse_args([
             'variables']))
+        cli.variables(self.parser.parse_args([
```
Comment (r39132) [inline tests/core.py]: For the lines below, can you add come comments to explain the tests?

File: tests/core.py [hunk 2]
```diff
@@ -906,7 +906,52 @@ def test_variables(self):
...
+            'variables', '-s', 'bar', 'updated']))
+        cli.variables(self.parser.parse_args([
+            'variables', '-s', 'foo', '{"foo":"oops"}']))
+        cli.variables(self.parser.parse_args([
+            'variables', '-x', 'foo']))
+        cli.variables(self.parser.parse_args([
+            'variables', '-i', 'variables1.json']))
+
+        assert models.Variable.get('bar') == 'original'
```
Comment (r39132) [inline tests/core.py]: How can bar be  'original' if it were set to 'updated'?

**Comment by r39132** [comment]
+1 after the code style (landscape) errors are fixed.

**Comment by criccomini** [comment]
@r39132 does this overlap with any existing CLI functionality? Thought I recall being able to set variables via CLI already (can't find it now though, but CLI is getting pretty complex).

**Comment by normster** [comment]
@criccomini currently the CLI is capable of manually setting and getting a variable. This adds functionality to import a file to set multiple at once, and also export values to a file.

**Comment by criccomini** [comment]
Ah, got it. Nice!

Random thought: a similar feature for connections would be interesting (though would require passwords to be in the file). There was some discussion on the mailing list about this.

**Comment by r39132** [comment]
@criccomini the true value of this functionality is that we can now set up variables using automation! With this CLI subcommand for the existing variables command, we can easily set up a new airflow services, exporting and importing variables. I agree the connection export/import would be the next useful piece to add.

**Comment by r39132** [comment]
FYI.. this was actually merged.. but was left open for some reason so I am closing it.

**Pull request closed without merging**